nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -179,6 +179,39 @@ Please see the [documentation on the `?depth` query parameter](../rest-api/overv
 
 ## UI, GraphQL, and REST API Filter Changes
 
+### Post Migration Helper Command `audit_dynamic_groups`
```

Comment:
This command should also be documented in `nautobot/docs/administration/nautobot-server.md` - maybe most of this content should be moved there and just linked to from this document?